src/main/java/org/gridsuite/modification/server/modifications/AbstractScaling.java

```diff
@@ -52,54 +49,15 @@ public void apply(Network network, Reporter subReporter) {
                 .filter(distinctByKey(FilterInfos::getId))
                 .collect(Collectors.toMap(FilterInfos::getId, FilterInfos::getName));
 
-        // export filters from filter server
-        String workingVariantId = network.getVariantManager().getWorkingVariantId();
-        UUID uuid = ((NetworkImpl) network).getUuid();
-        Map<UUID, FilterEquipments> exportFilters = filterService
-                .exportFilters(new ArrayList<>(filters.keySet()), uuid, workingVariantId)
-                .stream()
-                .peek(t -> t.setFilterName(filters.get(t.getFilterId())))
-                .collect(Collectors.toMap(FilterEquipments::getFilterId, Function.identity()));
-
-        // collect all filters with wrong equipments ids
-        Map<UUID, FilterEquipments> filterWithWrongEquipmentsIds = exportFilters.entrySet().stream()
-                .filter(e -> !CollectionUtils.isEmpty(e.getValue().getNotFoundEquipments()))
-                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-
-        Boolean noValidEquipmentId = exportFilters.values().stream()
-                .allMatch(filterEquipments -> filterEquipments.getIdentifiableAttributes().isEmpty());
-
-        if (noValidEquipmentId) {
-            String errorMsg = scalingInfos.getErrorType() + ": There is no valid equipment ID among the provided filter(s)";
-            createReport(subReporter, "invalidFilters", errorMsg, TypedValue.ERROR_SEVERITY);
+        Map<UUID, FilterEquipments> exportFilters = ModificationUtils.getInstance().getUuidFilterEquipmentsMap(filterService, network, subReporter, filters, scalingInfos);
+        if (exportFilters == null) {
             return;
```

Comment:
Maybe instead of if (exportFilters == null) return;, we change it to 
if (exportFilters != null) {
    // remaining code 
}